### PR TITLE
support Polygons and LineStrings via their bbox

### PR DESCRIPTION
**Pull request by andrewharvey**

### CODE REVIEW

**Comment by andrewharvey** [comment]
This doesn't merge features across tile boundaries, so you can end up with multiple accessible elements for each feature.

I considered using a feature `id` if exists to give strictly 1 accessible element per feature, but it might make sense to see if this get's addressed in GL JS https://github.com/mapbox/mapbox-gl-js/issues/6019

File: example/index.js [hunk 1]
```diff
@@ -13,21 +13,97 @@ div.style.bottom = 0;
...
   container: div,
   style: 'mapbox://styles/mapbox/streets-v9',
-  center: [-79.387, 43.656],
-  zoom: 13.5
+  center: [-93.87, 41.65],
+  zoom: 3
 });
 
 map.on('load', () => {
+  map.addSource('data', {
```
Comment (tristen) [inline example/index.js]: Can this be split into its own example instead? Demonstrating adding a custom source and applying accessibility to it.

**Comment by tristen** [comment]
Sweet!

Can the changes added to `example.index.js` be split into its own example instead? Demonstrating adding a custom source and applying accessibility to it.

**Comment by andrewharvey** [comment]
> Can the changes added to example.index.js be split into its own example instead? Demonstrating adding a custom source and applying accessibility to it.

Sure. I think at some point it makes sense to move the examples into `mapbox-gl-js` as it ensures they are more easily discoverable and we leverage all the existing infrastructure, re hosting, easy viewing online, auto access token.

**Pull request merged**